src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -164,6 +164,51 @@ def find_subscriptions_on_login(self,
         # use deepcopy as we don't want to persist these changes to file.
         return deepcopy(consolidated)
 
+    def find_subscriptions_in_cloud_console(self, tokens):
+        from datetime import datetime, timedelta
+        import jwt
+        arm_token = tokens[0]  # cloud shell gurantees that the 1st is for ARM
+        arm_token_decoded = jwt.decode(arm_token, verify=False, algorithms=['RS256'])
+        tenant = arm_token_decoded['tid']
+        user_id = arm_token_decoded['unique_name'].split('#')[-1]
+        subscription_finder = SubscriptionFinder(self.auth_ctx_factory, None)
+        subscriptions = subscription_finder.find_from_raw_token(tenant, arm_token)
+        consolidated = Profile._normalize_properties(user_id, subscriptions, False)
```

Comment:
Add parameter name for the third parameter (the boolean) to improve readability.